tests/resources/dspa-lite.yaml

```diff
@@ -9,6 +9,7 @@ spec:
     deploy: true
     enableOauth: false
     enableSamplePipeline: true
+    pipelineStorage: kubernetes
```

Comment:
As discussed in Slack, could you please make a separate DSPA and a separate test function in `.github/scripts/tests/tests.sh` that just tests your new integration test? That way we keep the existing test coverage as well.